.devcontainer/Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN apt-get update \
         golang.org/x/lint/golint \
         github.com/alecthomas/gometalinter \
         honnef.co/go/tools/... \
-        github.com/golangci/golangci-lint/cmd/golangci-lint \
+        #github.com/golangci/golangci-lint/cmd/golangci-lint \
```

Comment:
what was the issue here? doesn't work anymore?

Reply:
I think it's a dynamic package getting downloaded into the container, and it started to error out recently. So, I had to disable it to make the container build process work. Justin sent out this full fix PR: https://github.com/Azure/azure-service-operator/pull/1059